tools/server/webui/src/routes/+layout.svelte

```diff
@@ -155,7 +156,10 @@
 />
 
 <Sidebar.Provider bind:open={sidebarOpen}>
-	<div class="flex h-screen w-full">
+	<div
+		class="flex h-screen w-full"
+		style:height={`${innerHeight}px`}
```

Comment:
```suggestion
		style:height="{innerHeight}px"
```

This is always easier to read :)